app/services/planned_disbursement_history.rb

```diff
@@ -55,16 +55,19 @@ def revise_entry(entry, value)
 
     new_entry.update!(
       planned_disbursement_type: :revised,
-      value: value
+      value: value,
+      report: current_report,
     )
   end
 
   def update_entry(entry, value)
     entry.update!(value: value)
   end
 
-  def record_history?
-    !@activity.organisation.service_owner?
+  def current_report
+    unless @activity.organisation.service_owner?
```

Comment:
I wonder if `Organisation` needs a `delivery_partner?` method so this could be `if @activity.organisation.delivery_partner?`, especially since this is the more common option.

Reply:
I wasn't sure if the categories "not a service owner" and "delivery partner" were the same tthing. If they are, I'm happy to add this method. I think the intent here is that reports are required from any organisation that isn't BEIS, and therefore needs their spending reviewed by BEIS.